WordPress/src/main/res/values/stats_styles.xml

```diff
@@ -237,4 +242,9 @@
         <item name="android:ellipsize">end</item>
         <item name="android:maxLines">1</item>
     </style>
+
+    <style name="StatsNewTag" parent="TextAppearance.AppCompat.Body2">
```

Comment:
Since `stats_block_tag_item` is pretty generic, should we rename this to just `StatsTag`? Maybe there will be other tags in the future.

Reply:
you're right!